### ISSUE TITLE
Material 3 support

### ISSUE THREAD

**Issue by mainrs** [KevinnZou/compose-swipebox-multiplatform#2]
Hello!

Having this library rely on material3 instead of material would allow for proper theming functionality (like dynamic themes) to work properly!

Thank you for the great library!

**Comment by KevinnZou** [comment]
@mainrs Thanks for your suggestion! I will upgrade it to Material3 when I am back from vacation.

**Comment by mainrs** [comment]
Thank you!

**Comment by KevinnZou** [comment]
@mainrs After researching, I discovered that the `Modifier.swipeable` we are using in this library is from Material2 and is not available in Material3. Therefore, unfortunately, we may not be able to upgrade to it currently.

**Comment by mainrs** [comment]
The API has actually been renamed and moved into foundation: 
https://developer.android.com/jetpack/compose/touch-input/pointer-input/migrate-swipeable

**Comment by KevinnZou** [comment]
Yes, we also noticed that and have just migrated to anchoredDraggable for the [Android version](https://github.com/KevinnZou/compose-swipeBox/releases/tag/1.3.0) of this library. We will also do it for the multiplatform version once Compose 1.6.0 is stable.
